go.mod

```diff
@@ -136,9 +136,13 @@ require (
 	github.com/cespare/xxhash/v2 v2.3.0 // indirect
 	github.com/chai2010/gettext-go v1.0.2 // indirect
 	github.com/checkpoint-restore/go-criu/v5 v5.3.0 // indirect
-	github.com/cilium/ebpf v0.9.1 // indirect
-	github.com/containerd/console v1.0.3 // indirect
-	github.com/containerd/ttrpc v1.2.2 // indirect
+	github.com/cilium/ebpf v0.11.0 // indirect
+	github.com/containerd/cgroups/v3 v3.0.3 // indirect
+	github.com/containerd/console v1.0.4 // indirect
+	github.com/containerd/containerd/api v1.7.19 // indirect
+	github.com/containerd/errdefs v0.1.0 // indirect
```

Comment:
Notice that there is a WIP PR to revert the change for `containerd/errdefs`: https://github.com/containerd/containerd/pull/10712. 

IIUC, there is a dep circle. See https://github.com/containerd/containerd/pull/10693 "containerd/errdefs to v0.2.0" and https://github.com/microsoft/hcsshim/pull/2269 for more details.

Reply:
@pacoxu ACK, we can adapt/adopt as things land and tags get cut in different places. our usage of containerd/api on the client side is very limited, so as long as our CI jobs hold up, we should be relatively fine. 🤞🏾